e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -246,23 +246,23 @@
         "eventHubEndpoints": {
           "events": {
             "retentionTimeInDays": 1,
-            "partitionCount": 4
+            "partitionCount": 10
           }
         },
         "cloudToDevice": {
           "defaultTtlAsIso8601": "PT1H",
-          "maxDeliveryCount": 10,
+          "maxDeliveryCount": 100,
           "feedback": {
             "ttlAsIso8601": "PT1H",
-            "lockDurationAsIso8601": "PT60S",
-            "maxDeliveryCount": 10
+            "lockDurationAsIso8601": "PT5S",
+            "maxDeliveryCount": 100
```

Comment:
The max delivery count property on the IoT hub determines the maximum number of times a message can transition between the Enqueued and Invisible states. After that number of transitions, the IoT hub sets the state of the message to Dead lettered.